.travis.yml

```diff
@@ -26,6 +26,7 @@ before_cache:
 scala:
   - 2.11.12
   - 2.12.8
+  - 2.13.0-RC3
```

Comment:
It's time to target 2.13.0

Reply:
I wish it were, but I'm still blocked on (at least) https://github.com/FasterXML/jackson-module-scala/pull/417